http/src/request/channel/invite/create_invite.rs

```diff
@@ -62,19 +72,29 @@ impl<'a> CreateInvite<'a> {
     ///
     /// If no age is specified, Discord sets the age to 86400 seconds, or 24 hours.
     /// Set to 0 to never expire.
-    pub fn max_age(mut self, max_age: u64) -> Self {
+    pub fn max_age(mut self, max_age: u64) -> Result<Self, CreateInviteError> {
+        if max_age > 604800 {
```

Comment:
As shown in the [`clippy` CI check](https://github.com/twilight-rs/twilight/runs/1940840243?check_suite_focus=true#step:5:362), consider changing `604800` to `604_800`.